packages/schematics/update/update/npm.ts

```diff
@@ -61,17 +61,20 @@ function getNpmConfigOption(
   return value;
 }
 
-function getNpmClientSslOptions(strictSsl?: string, cafile?: string) {
-  const sslOptions: { strict?: boolean, ca?: Buffer } = {};
+function getNpmClientSslOptions(
+  strictSsl?: string | boolean | undefined,
+  cafile?: string | boolean | undefined,
+) {
+  const sslOptions: { strict?: boolean; ca?: Buffer } = {};
 
-  if (strictSsl === 'false') {
-    sslOptions.strict = false;
-  } else if (strictSsl === 'true') {
+  if (strictSsl === undefined) {
     sslOptions.strict = true;
+  } else {
+    sslOptions.strict = <boolean> strictSsl;
```

Comment:
if the `strictSsl` value is either a boolean or undefined then a direct assignment to `sslOptions.strict` should be the only change need (excepting the parameter type change for the function).  The entire conditional structure can be removed.  The underlying library defaults to `true` if the value is not set or undefined.

Reply:
Done :+1: